xrmocap/data/dataset/mvp_dataset.py

```diff
@@ -219,11 +218,25 @@ def __getitem__(
 
         image = [img for img in mview_img_tensor]
 
-        return image, meta, skip
+        return image, meta
 
     def __len__(self):
         return super().__len__()
 
+    def get_one_frame(self, index):
+        skip = 0
```

Comment:
`skip_no_person=False`?

If it is only true or false, bool is better. And it is more readable if we tell users why we skip it, e.g., [skip_no_iphone](https://github.com/open-mmlab/mmhuman3d/blob/21a8ce76a0e581e514f31cf31ff910bbb2d417be/mmhuman3d/data/data_converters/humman.py#L30)

Reply:
Fixed in [44f2617](https://github.com/openxrlab/xrmocap/pull/18/commits/44f2617631d80f41de6a41827f60259f4e154478).